culinary-canvas/capstone-project/backend/index.js

```diff
@@ -252,17 +509,27 @@ app.get('/recipes', async (req, res) => {
             acc[like.recipeId] = like._count.recipeId;
             return acc;
         }, {})
-        data.hits.forEach(hit => {
-            hit.likes = likesMap[hit.recipe.uri] || 0;
-        });
 
-        return res.json(data.hits)
+        const recipesWithHealthScores = data.hits.map(hit => {
+            const recipe = hit.recipe;
+            const healthInfo =calculateHealthScore(recipe, userPreferences);
+
+            return {
+                ...recipe,
+                likes:likesMap[hit.recipe.uri] || 0,
+                healthScore : healthInfo ? healthInfo.score : 0,
+                healthColor : healthInfo ? healthInfo.color : 0
+            }
```

Comment:
does this mean the same recipe could have different health scores for different users based on their diet and allergens?